lib/tasks/db.rake

```diff
@@ -0,0 +1,42 @@
+require_relative '../straight-server'
+
+namespace :db do
+  task :environment do
+    include StraightServer::Initializer
+    ConfigDir.set!
+    create_config_files
+    read_config_file
+    connect_to_db
+  end
+
+  desc "Migrate the database"
```

Comment:
For the future: note your next description says "Rollbacks the database" (a description of action), but this one is a command ("Migrate!"). Consistency is important: probably should be "Migrates the database". I'll fix it myself for now.